src/app/option/option.component.html

```diff
@@ -1,16 +1,17 @@
-<h2>Option</h2>
+<h2>Options</h2>
 <fieldset>
   <legend>Choose format:</legend>
 
   <form class="stackoverflow" [formGroup]="form">
     <div>
       <input type="radio" id="symbols-and-chars" formControlName="symbols" [value]="symbolsFormats.SymbolsAndCharacters"
              checked>
-      <label for="symbols-and-chars">Symbols and Characters (<kbd>⇧ Shift</kbd>&nbsp;<kbd>⌘ Command</kbd>&nbsp;<kbd>⌫ Delete</kbd>…)</label>
+      <label for="symbols-and-chars">Symbols and Characters (<kbd>⇧ Shift</kbd>&nbsp;<kbd>⌘ Command</kbd>&nbsp;<kbd>⌫
+          Delete</kbd>…)</label>
```

Comment:
Sorry, I’m trying to work out why this line was split… Might be something to do with my VSCode setup: I have Prettier installed. Will see if I can get it to stop doing this and will do another commit.